### PR TITLE
Sync #491 with Indonesian

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

**Review by KnorpelSenf — APPROVED** [review]

**Comment by fadzikri** [comment]
umm... what should i review @roj1512 ?

File: site/docs/id/hosting/deno-deploy.md [hunk 1]
```diff
@@ -20,27 +20,25 @@ Hasil dari tutorial disini dapat dilihat di [repositori bot kami](https://github
 2. Buat sebuah file dengan nama `mod.ts` atau `mod.js`, ataupun nama lainnya sesuai dengan keinginanmu (tetapi kamu harus mengingatnya karena nanti file tersebut akan digunakan sebagai file deploy utama). File tersebut berisikan:
 
 ```ts
-import { webhookCallback } from "https://deno.land/x/grammy/mod.ts";
-import { serve } from "https://deno.land/x/sift@0.5.0/mod.ts";
+import { serve } from "https://deno.land/std/http/server.ts";
+import { Bot, webhookCallback } from "https://deno.land/x/grammy/mod.ts";
```
Comment (CikiMomogi) [inline site/docs/id/hosting/deno-deploy.md:24]: Shouldn't this be `import { webhookCallback } from "https://deno.land/x/grammy/mod.ts";`?

Comment (CikiMomogi) [reply site/docs/id/hosting/deno-deploy.md:24]: I will make the pull request for it.

**Comment by roj1512** [comment]
@fadzikri Nothing here, it was already reviewed.

**Pull request merged**